draft-ietf-idr-5g-edge-service-metadata-24.xml

```diff
@@ -962,6 +1054,7 @@
           octets, excluding the sub-Type and the length field. The Length is 6
           for the AS-Scope Sub-Type.</t>
 
+	  <!-- XXX JMH need the usual text about the reserved value contentss. -->
           <t hangText="- Reserved (8 bits):">Reserved for future use.</t>
```

Comment:
will add one sentence in the Error Handling section indicating the Sub-TLV is ignored if the reserved bits/bytes are not same as specified by the document.

Reply:
For this and similar cases, you're defining the fields.  It's okay to use similar boilerplate in each of those places to say that RESERVED means something like the MUST set to zero by originators of this TLV in this document, MUST/SHOULD be ignored.

For the per field stuff, you wouldn't want to defer this to the error handling text because something that needs per field clarification adds a section or paragraph to the error handling for each referenced item.  So, there's clarity incentive to have it in the defining section for the TLV.

Reply:
Added the following statement to all Reserved bits: 

_"These bits are reserved for future use and MUST be set to zero"_